helm_deploy/values-preprod.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 4
 
 image:
-  repository: docker.io/mojdigitalstudio/whereabouts-api
+  repository: quay.io/hmppswhereabouts-api
```

Comment:
are these urls correct? missing a / ?

Reply:
agreed, outdated comment, fixed if you look at the source instead